### ISSUE TITLE
Can we use it in production?

### ISSUE THREAD

**Issue by chintan-mishra** [komamitsu/retrofit-converter-msgpack#1]

**Comment by chintan-mishra** [comment]
Below I have linked the updated Wiki on Retrofit repository. This update adds `retrofit-converter-msgpack` as one of the Third Party Converters

https://github.com/square/retrofit/wiki/Converters

**Comment by komamitsu** [comment]
Yeah. We've been using this library in staging w/o any problem and it's almost ready for production. Actually, I'm using msgpack-java:0.8.13 with this library for testing in staging, but I believe this library works with the latest msgpack-java (0.8.16)
